systemd/systemd.go

```diff
@@ -834,10 +834,9 @@ func (s *systemd) Restart(serviceName string, timeout time.Duration) error {
 	if s.mode == GlobalUserMode {
 		panic("cannot call restart with GlobalUserMode")
 	}
-	if err := s.Stop(serviceName, timeout); err != nil {
-		return err
-	}
-	return s.Start(serviceName)
+	// XXX: timeout is ignored
```

Comment:
I'm not sure what the "`XXX`" means in the context of a comment; is it something you plan to do before removing the "draft" flag from the PR?

Reply:
No, it's about timeout being passed to Restart() as mandated by our systemd interface definition, but getting ignored here.

Reply:
So, it's just to say that we are aware of the timeout, but that we cannot implement it?

If there is no way to implement the timeout mechanism in this case, maybe we should have a 
```
// TODO: remove unused timeout parameter from the interface
```

instead?